package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -91,30 +92,54 @@ def load_data(
         A tuple containing the data catalog and the pipeline dictionary
         and the session store.
     """
-    if package_name:
-        configure_project(package_name)
-    else:
-        # bootstrap project when viz is run in dev mode
-        bootstrap_project(project_path)
-
-    with KedroSession.create(
-        project_path=project_path,
-        env=env,
-        save_on_close=False,
-        extra_params=extra_params,
-    ) as session:
-        # check for --include-hooks option
-        if not include_hooks:
-            session._hook_manager = _VizNullPluginManager()  # type: ignore
-
-        context = session.load_context()
-        session_store = session._store
-        catalog = context.catalog
-
-        # Pipelines is a lazy dict-like object, so we force it to populate here
-        # in case user doesn't have an active session down the line when it's first accessed.
-        # Useful for users who have `get_current_session` in their `register_pipelines()`.
-        pipelines_dict = dict(pipelines)
-        stats_dict = _get_dataset_stats(project_path)
+    if is_lite:
+        # [TODO: Confirm on the context creation]
+        context = KedroContext(
+            package_name="{{ cookiecutter.python_package }}",
+            project_path=project_path,
+            config_loader=OmegaConfigLoader(conf_source=str(project_path)),
+            hook_manager=_VizNullPluginManager(),
+            env=env,
+        )
+
+        # [TODO: Confirm on the session store creation]
+        session_store = None
```

Comment:
couple of things to consider:
- `settings.py` are not read because we are skipping `bootstrap_project` or `configure_project`. See https://docs.kedro.org/en/stable/kedro_project_setup/session.html#bootstrap-project-and-configure-project . We need to read `settings.py` to ensure `CONFIG_LOADER_ARGS` are read.
- OmegaConfigLoader need to takes argument. In framework settings it's usually `OmegaConfigLoader(conf_source="conf", base_env="base", default_run_env="local")`, but as mentioned above we need to take the settings from `settings.py` if provided. Otherwise use the default as described.